src/OmniSharp.Shared/Options/RoslynExtensionsOptions.cs

```diff
@@ -11,7 +11,7 @@ public class RoslynExtensionsOptions : OmniSharpExtensionsOptions
         public bool EnableAnalyzersSupport { get; set; }
         public bool EnableImportCompletion { get; set; }
         public bool EnableAsyncCompletion { get; set; }
-        public int DocumentAnalysisTimeoutMs { get; set; } = 30 * 1000;
+        public int DocumentAnalysisTimeoutMs { get; set; } = 80 * 1000;
```

Comment:
I don't think this should be changed as part of this PR unless there is a compelling reason. 

```suggestion
        public int DocumentAnalysisTimeoutMs { get; set; } = 30 * 1000;
```

Reply:
Yes, that was adhoc raw solution to fix timeout in one test.

Reply:
After your suggestions problem is solved.